CONTRIBUTING.md

```diff
@@ -76,3 +78,131 @@ A PR cannot be merged as long as there are any failing tasks.
     - Please use the shorthand `#123` and not the full URL.
 
 If you do not meet any of these guidelines, don't fret. Chances are they will be fixed upon rebasing but please do try to meet them to remove some of the workload.
+
+
+## How do I add a new feature?<a name="first-contrib"></a>
```

Comment:
What's up with the anchor tag here?

Reply:
I think it was initially used elsewhere but then removed.